content/en/tracing/connect_logs_and_traces/python.md

```diff
@@ -64,10 +69,14 @@ def tracer_injection(logger, log_method, event_dict):
     trace_id, span_id = get_correlation_ids()
 
     # add ids to structlog event dictionary
-    # if no trace present, set ids to 0
     event_dict['dd.trace_id'] = trace_id or 0
     event_dict['dd.span_id'] = span_id or 0
 
+    # add the env, service, and version configured for the tracer
+    event_dict['dd.env'] = config.Config().env or ""
+    event_dict['dd.service'] = config.Config().service or ""
+    event_dict['dd.version'] = config.Config().version or ""
```

Comment:
```suggestion
    event_dict['dd.env'] = ddtrace.config.env or ""
    event_dict['dd.service'] = ddtrace.config.service or ""
    event_dict['dd.version'] = ddtrace.config.version or ""
```

Reply:
The `.Config()` part is not needed.

If you decide not to accept my other suggestion for the import above, this would just need to be `config.env or ""`, etc without the `ddtrace.` prefix